host/sgx/sgxload.c

```diff
@@ -134,7 +134,12 @@ static sgx_secs_t* _new_secs(
         return NULL;
 
     memset(secs, 0, sizeof(sgx_secs_t));
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+    secs->size = 68719476736; /* Max acceptable */
```

Comment:
This is the maximum size SGX allows. When signing base images we don't know the size that the final image will have, so we chose the maximum here. If we want to be able to use any base image for EEID, this is needed. If we decide that we want EEID to work only with special base images (like zero memory images), we can make this conditional on a flag.

Reply:
move the github comment into a comment in the code?

Reply:
I figured this one is going to trigger a bit of discussion, so I started the discussion right then. I'll copy it into the code too.

Reply:
(Done.)